editions/tw5.com/tiddlers/wikitext/Wikitext Cheat-Sheet.tid

```diff
@@ -0,0 +1,143 @@
+created: 20220821184022101
+modified: 20221010130927409
+tags: WikiText
+title: Wikitext Cheat-Sheet
+type: text/vnd.tiddlywiki
+
+\define extLink(text tid) [ext[$text$|https://tiddlywiki.com/#$tid$]]
+\define link(text tid) [[$text$|$tid$]]
+
+\define exportFilter()
+[[$(me)$]] [tag[$(me)$]]
+\end
+
+\define exportMeButton(btnText:"Export Me & Tagged")
+\whitespace trim
+<$set name="me" value=<<currentTiddler>> >
+  <$tiddler tiddler="$:/core/templates/exporters/JsonFile">
+    <$button class="tc-btn-download">
+      <$action-sendmessage $message="tm-download-file" $param=<<currentTiddler>> exportFilter=<<exportFilter>> filename={{{ [<me>addsuffix{!!extension}] }}}/>
+      <$text text="""$btnText$"""/>
+    </$button>
+  </$tiddler>
+</$set>
+\end
+
+\define .xL(text tid)
+\whitespace trim
+<$set name="link" value={{{ [[$tid$]is[tiddler]then<link """$text$""" """$tid$""">else<extLink """$text$""" """$tid$""">] }}}>
+<<link>>
+</$set>
+\end
+
+\define .t(tid) <$transclude tiddler="""$tid$""" mode=block/>
+\define .cb(title) <$codeblock code={{$title$}}/>
+
+<!-- Helper macros. The "old" wikitext style is intentional. The code should be compatible with TW v5.1.15 -->
+<!-- extLink() and link() create the external and internal links for the .xL macro -->
+<!-- exportFilter() defined the filter for the tids to be exported -->
+<!-- exportMeButton() saves this tiddler and all it's dependencies into a JSON export file -->
+<!-- .xL ... creates an internal link, if tiddler exists otherwise it creates a link to tiddlywiki.com -->
+<!-- .t, .cb are macros that that transclude and show code blocks from example tiddlers -->
+
+!! Overview
+
+The following table contains the basic ~TiddlyWiki wikitext features. The "Cheat-Sheet" and all dependencies can be exported using the button below.
+
+<<exportMeButton "Export Cheat-Sheet">>
+
+!! Cheat Sheet
+
+| Example                        | Example Markup                 |h
+| how the example markup renders | what to enter as tiddler text  |
+|>| ! <<.xL "Basic Text Formatting" "Formatting in WikiText">>    |
+|''bold''                        |`''bold''` (tow single hyphens) |
```

Comment:
"tow" is a typo.

I think the entire comment `(two single hyphens)` doesn't belong here. Either we add another column for "Notes" or we should drop it.